### PR TITLE
GameDB: Add GIF FIFO hack for Spongebob: Creature From the Krusty Krab

### PR DESCRIPTION
**Pull request by MrCK1**
This PR adds the GIF FIFO hack for Spongebob Creature From the Krusty Krab to fix bad graphics. 

Before:
pcsx2_2018_03_04_11_51_22_746

After:
pcsx2_2018_03_04_11_51_09_375

### CODE REVIEW

File: bin/GameIndex.dbf [hunk 1]
```diff
@@ -68,6 +68,7 @@
 -- DMABusyHack       = 1 // Mana Khemia, Metal Saga. Denies writes to the DMAC when it's busy.
 -- VIFFIFOHack       = 1 // Transformers Armada, Test Drive Unlimited. Fixes slow booting issue.
 -- VIF1StallHack     = 1 // SOCOM II HUD and Spy Hunter loading hang.
+-- GIFFIFOHack		 = 1 // Enables the GIF MFIFO. Needed for Wallace & Grommit, Hot Wheels, DJ Hero.
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: Extra whitespace.

**Comment by FlatOutPS2** [comment]
You forgot to add it for `SLPM-66694`

Note; You left out `Squarepants` in the PR/commit title.

**Comment by MrCK1** [comment]
@FlatOutPS2 Thanks, I didn't see that because it's labeled incorrectly as `Spongebob`

**Comment by FlatOutPS2** [comment]
> @FlatOutPS2 Thanks, I didn't see that because it's labeled incorrectly as Spongebob

Actually, Spongebob is the subtitle of the game, so it makes sense. The full title is `Spongebob Squarepants: Spongebob`, but it's usually just called Spongebob.

File: bin/GameIndex.dbf [hunk 2]
```diff
@@ -27489,6 +27491,7 @@ Region = NTSC-J
 Serial = SLPM-66694
 Name   = Spongebob
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: You forgot to change the name here as well.

Comment (MrCK1) [reply bin/GameIndex.dbf]: No, I just reverted that because adding the full title would be incorrect for this region.

Comment (lightningterror) [reply bin/GameIndex.dbf]: It might get confusing with other Spongebob titles this way. I don't know how many there are.

Comment (FlatOutPS2) [reply bin/GameIndex.dbf]: It's the only Japanese Spongebob title. Japanese people will recognise the game with the current name (the Japanese cartoon title also lacks Squarepants).

File: bin/GameIndex.dbf [hunk 3]
```diff
@@ -68,6 +68,7 @@
 -- DMABusyHack       = 1 // Mana Khemia, Metal Saga. Denies writes to the DMAC when it's busy.
 -- VIFFIFOHack       = 1 // Transformers Armada, Test Drive Unlimited. Fixes slow booting issue.
 -- VIF1StallHack     = 1 // SOCOM II HUD and Spy Hunter loading hang.
+-- GIFFIFOHack       = 1 // Enables the GIF MFIFO. Needed for Wallace & Grommit, Hot Wheels, DJ Hero.
```
Comment (FlatOutPS2) [inline bin/GameIndex.dbf]: `Enables the GIF FIFO.`

**Pull request merged**